modules/saml/lib/SP/LogoutStore.php

```diff
@@ -407,6 +400,7 @@ public static function logoutSessions($authId, $nameId, array $sessionIndexes)
         // serialize and anonymize the NameID
         // TODO: remove this conditional statement
         if (is_array($nameId)) {
+            /** @psalm-suppress UndefinedMethod */
```

Comment:
This one is also suppressing a real bug.

Reply:
I disagree. Psalm runs on PHP7.3 with composer update, so using the 4.x branch of saml2